src/common/Pressable/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Pressable as RNPressable } from 'react-native';
+
+import styles from './styles';
+import { ButtonProps } from './types';
+
+const Pressable: React.FunctionComponent<ButtonProps> = ({
+  containerStyle,
+  disabledStyle,
+  disabled = false,
+  children,
+  ...props
+}) => (
+  <RNPressable
+    accessibilityRole="button"
+    style={({ pressed }) => [
+      pressed ? styles.pressed : styles.notPressed,
+      styles.commonContainer,
+      containerStyle,
+      disabled ? disabledStyle : {},
+    ]}
+    disabled={disabled}
+    {...props}>
+    {children}
```

Comment:
I think it's not needed to explicitly render the `children` here. It'd be included in `props` if we remove the [`destructuring` of it at line 11](https://github.com/rootstrap/react-native-base/pull/212/files#diff-5b89338e4381280d278ab9f6f5013ea37deef797f755684209268a99413703f6R11).